app/controllers/storage_locations_controller.rb

```diff
@@ -11,7 +11,7 @@ def quantity
   def index
     @inventory = View::Inventory.new(current_organization.id)
     @selected_item_category = filter_params[:containing]
-    @items = StorageLocation.items_inventoried(current_organization, @inventory)
+    @items = @inventory.items_for_select
```

Comment:
Moved this method, as it doesn't really have anything to do with storage locations.

Reply:
This seems totally unrelated to the rest of this PR?

Reply:
It is unrelated, I can remove it out of this PR if preferred. But I figured it's a small enough change (method is only used in this one place), that it's not worth a separate PR.

Reply:
Please move it out of the PR. Small PRs are fine, or you can collect refactors like this into a separate "refactor" PR (assuming there aren't more than a handful in it).

Reply:
Removed